lib/osf-components/addon/components/carousel/component.ts

```diff
@@ -49,4 +61,104 @@ export default class Carousel extends Component {
         this.carouselItems[activeIndex].set('isActive', false);
         this.carouselItems[newIndex].set('isActive', true);
     }
+
+    /** registerKeyboard() is a function that enables users to navigate certain
+     *  components with a keyboard rather than using a mouse. The keybindings are
+     *  designed for optimal ergonomic comfort as well as improving overall user
+     *  experience while within the application.
+     *
+     *  @author : chthonix
+     *  @date : Sat Oct 01 2022 14:26:06 GMT-0400
+     */
+    registerKeyboard() {
```

Comment:
There are a lot of things going on in this method. First of all, for some reason I have still yet to decipher, this method would only be invoked when you tab focus to the carousel element. When that happens, it will attach an event handler to the root document to catch all `keydown` events. The purpose of the event handler is to enable users to navigate the carousel using `ctrl+f` and `ctrl+j`, among other things. However, the event handler is not removed when you focus away from the carousel element.

There are several side effects here:

1. Because `event.preventDefault()` is invoked by the event handler, that means any browser default keyboard shortcut that use a combination of `alt` or `ctrl` keys with other keys will be disabled after the event handler is attached. That means, after you tab focus to the carousel element, `ctrl+f` can no longer find you text on the current page, which is the most common browser default shortcut.
2.  Because the event handler catches all `keydown` events, any keyboard input that is not in combination with `alt` or `ctrl` will make toastr toast an error. The reason this doesn't really toast any error is that there seems to be something wrong when invoking the `intl` service.
3. Because the event handler attached to the root document is never removed, that means the handler is still active when you transition to another ember page. That means browser default shortcuts will be unusable on other ember pages.

Also, just out of curiosity, are the keystroke combinations defined here for navigating the carousel a universally accepted standard for a11y? How would user know that `ctrl+f` and `ctrl+j` (and other combinations defined here) can control the interactive elements of the carousel?